depthai_ros_driver/include/depthai_ros_driver/conversion.hpp

```diff
@@ -0,0 +1,61 @@
+#pragma once
+
+#include <depthai_datatype_msgs/RawImgFrame.h>
+#include <depthai_datatype_msgs/RawNNData.h>
+#include <sensor_msgs/Image.h>
+#include <cv_bridge/cv_bridge.h>
+#include <sensor_msgs/image_encodings.h>
+#include <opencv2/opencv.hpp>
+#include <ros/console.h>
+
+namespace rr {
+
+template <typename T>
+struct adapt_dai2ros {
+    using InputType = T;
+    using OutputType = T;
```

Comment:
Remove const, volatile and reference from T? Might be useful to use `remove_cv_t` and `remove_reference_t` on `T`